core/cmd.py

```diff
@@ -255,6 +271,30 @@ async def togglescreen(self):
         self.logging.info("toggling screen")
         await self.bluetooth.send(Common().toggleScreenFreeze())
 
+    async def set_brightness(self, argument: str) -> None:
+        """sets the brightness of the screen"""
+        try:
+            conv_brightness = int(argument)
+            if conv_brightness in range (5, 101):
+                self.logging.info(f"setting brightness of the screen: {argument}%")
+                await self.bluetooth.send(Common().set_screen_brightness(brightness_percent=conv_brightness))
+            else:
+                self.logging.error("brightness out of range")
+        except ValueError:
+            self.logging.error(f"Invalid integer: {argument}")
+                    
+    async def set_password(self, argument: str) -> None:
+        """sets connection password"""
+        try:
+            conv_password = int(argument)
+            if len(argument) == 6 and conv_password in range(0, 1000000):
+                self.logging.info(f"setting password: {argument}")
+                await self.bluetooth.send(Common().set_password(conv_password))
+            else:
+                self.logging.error(f"Password should be 6 digits long and in range 000000...999999")
+        except ValueError:
+            self.logging.error(f"Invalid integer: {argument}")
```

Comment:
We should either agree on using

`f"{argument}"`

or 

`"{}".format()`

but not both.

I really like **.format** because it provides a better readability at least for me because when using multiple aguments with longer names one could simply split them into multiple lines with ease.

Anyhow, I am fine with both but not mixed up.

Reply:
Disagree - the former provides a better readibility IMO, especially on a multiline statement.

```
print(f"var1 = {var1}, var2 = {var2} and the sum is {var1 + var2}"

# vs

print("var1 = {}, var2 = {} and the sum is {}".format(var1, var2, var1+var2))
```

Reply:
Okay, fair enough! We'll keep it and maybe I have to change my code style then :)